src/lib.rs

```diff
@@ -86,6 +86,26 @@ pub struct Metadata {
 /// ```
 #[macro_export]
 macro_rules! setup_panic {
+  ($passthrough:expr, $meta:expr) => {
+    #[allow(unused_imports)]
+    use std::panic::{self, PanicInfo};
+    #[allow(unused_imports)]
+    use $crate::{handle_dump, print_msg, Metadata};
+
+    #[cfg(not(debug_assertions))]
```

Comment:
❓ What do you think about having this logic baked in rather than in the closure itself? This seems right for our specific "don't log to firebase from debug builds," but it seems like this might be something we use/abuse more and more over time, and we may want to control what runs when from elsewhere.

Reply:
I'm not sure I understand what you're suggesting. You want to rip out the `debug_assertions` compile attribute?